apps/beeai-sdk/src/beeai_sdk/a2a/extensions/services/llm.py

```diff
@@ -52,17 +55,30 @@ class LLMServiceExtensionParams(pydantic.BaseModel):
     """Model requests that the agent requires to be provided by the client."""
 
 
+class LLMServiceExtensionSpec(BaseExtensionSpec[LLMServiceExtensionParams]):
+    URI: str = "https://a2a-extensions.beeai.dev/services/llm/v1"
+
+    @classmethod
+    def single_demand(
+        cls, name: str | None = None, description: str | None = None, suggested: tuple[str, ...] = ()
+    ) -> Self:
+        return cls(
+            params=LLMServiceExtensionParams(
+                llm_demands={name or "llm": LLMDemand(description=description, suggested=suggested)}
```

Comment:
IMO this sounds better

```suggestion
                llm_demands={name or "default": LLMDemand(description=description, suggested=suggested)}
```